python/examples/core/compilation.py

```diff
@@ -137,8 +137,10 @@ def emit_benchmarking_function(name: str,
     loop = scf.ForOp(zero, n_iterations, one, iter_args)
     with InsertionPoint(loop.body):
       start = std.CallOp(nano_time, [])
-      call = std.CallOp(
-          func, wrapper.arguments[:-num_results - 1] + loop.inner_iter_args)
+      print(f"loop inner iter args: {loop.inner_iter_args[0]}")
```

Comment:
Leftover debug spew

Reply:
Good catch, I'll drop it once the CI passes since it takes a long while.